tests/unit/processor/field_manager/test_field_manager.py

```diff
@@ -209,7 +209,7 @@
             "field3": ["value5", "value6", "value4"],
             "new_field": ["i exist"],
         },
-        {"new_field": ["i exist", "value1", "value2", "value3", "value4", "value5", "value6"]},
+        {"new_field": ["i exist", "value1", "value2", "value3", "value5", "value6", "value4"]},
```

Comment:
same but different. duplicates could be resolved in first comes first serves manner:

```suggestion
        {"new_field": ["i exist", "value1", "value2", "value3", "value5", "value4", "value6",]},
```